### PR TITLE
only display external reg number if not foreign company

### PR DESCRIPTION
**Pull request by ethatcher83**
only display external reg number if not foreign company

Resolves: PCI-111

### CODE REVIEW

File: document-generator-company-report/src/main/resources/company-report.html [hunk 1]
```diff
@@ -109,22 +109,25 @@ <h2 class="heading-large">Overview</h2>
         {{end}}
 
         {{if $companyRegistrationInformation.company_type.type}}
-            <li>Company type: {{$companyRegistrationInformation.company_type.type}}
-                {{if $companyRegistrationInformation.company_type.subtype}}/{{$companyRegistrationInformation.company_type.subtype}}{{end}}</li>
+        <li>Company type: {{$companyRegistrationInformation.company_type.type}}
```
Comment (Mathsh) [inline document-generator-company-report/src/main/resources/company-report.html]: formatting has been shifted, should be reverted.

File: document-generator-company-report/src/main/resources/company-report.html [hunk 2]
```diff
@@ -109,22 +109,25 @@ <h2 class="heading-large">Overview</h2>
...
+            {{if $companyRegistrationInformation.company_type.subtype}}/{{$companyRegistrationInformation.company_type.subtype}}{{end}}</li>
         {{end}}
 
-        {{if $companyRegistrationInformation.external_registration_number}}
+        {{if ne $companyRegistrationInformation.company_type.type "Overseas company"}}
+            {{if $companyRegistrationInformation.external_registration_number}}
             <li>Charity number: {{$companyRegistrationInformation.external_registration_number}}</li>
```
Comment (Mathsh) [inline document-generator-company-report/src/main/resources/company-report.html]: correct the formatting for this line.

File: document-generator-company-report/src/main/resources/company-report.html [hunk 3]
```diff
@@ -109,22 +109,25 @@ <h2 class="heading-large">Overview</h2>
...
+            {{end}}
         {{end}}
 
         {{if $companyRegistrationInformation.date_of_incorporation}}
-            <li>{{$companyRegistrationInformation.date_of_incorporation_label}}: {{$companyRegistrationInformation.date_of_incorporation}}</li>
+        <li>{{$companyRegistrationInformation.date_of_incorporation_label}}: {{$companyRegistrationInformation.date_of_incorporation}}</li>
```
Comment (Mathsh) [inline document-generator-company-report/src/main/resources/company-report.html]: revert the formatting

File: document-generator-company-report/src/main/resources/company-report.html [hunk 4]
```diff
@@ -109,22 +109,25 @@ <h2 class="heading-large">Overview</h2>
...
         {{end}}
 
         {{if $companyRegistrationInformation.status.company_status}}
-            <li>Status: {{$companyRegistrationInformation.status.company_status}}
-                {{if $companyRegistrationInformation.status.company_status_detail}}/{{$companyRegistrationInformation.status.company_status_detail}}{{end}}</li>
+        <li>Status: {{$companyRegistrationInformation.status.company_status}}
```
Comment (Mathsh) [inline document-generator-company-report/src/main/resources/company-report.html]: revert the formatting

**Comment by ethatcher83** [comment]
Resolved formatting issues

**Comment by leahtaylor** [comment]
@Mathsh issues addressed

**Pull request merged**